tests/matchers/test_equal.py

```diff
@@ -22,11 +22,17 @@ def test_register(self):
 
 
 class TestDictDiff:
-    def test_dict_diffs_with_different_length(self):
+    def test_dict_diffs_with_missing_key(self):
         d1 = {'a': 1}
         d2 = {'a': 1, 'b': 2}
```

Comment:
What happens if `d2 = {'a': 1, 'b': 2, 'c': 3}`? I guess the message will be the same, but is it better to list out all the keys that B has, but A does not? I guess it may complicate things more if the keys of 2 hashes are very different.

Reply:
I don't think it's better to list out all the keys as in 99% of cases I've seen only 1 or 2 keys are missing. Plus this is simpler to implement and maintain. If the diff is very complicated then just listing the keys out would not mean much anyway.